packages/react/src/components/Tooltip/Tooltip.base.tsx

```diff
@@ -68,6 +68,10 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
   }
 
   private _onRenderContent = (props: ITooltipProps): JSX.Element => {
-    return <p className={this._classNames.subText}>{props.content}</p>;
+    if (typeof props.content === 'string') {
+      return <p className={this._classNames.subText}>{props.content}</p>;
+    } else {
+      return <div>{props.content}</div>;
```

Comment:
You probably still want to keep the classname in case in here in case it is targeted directly:

```suggestion
      return <div className={this._classNames.subText}>{props.content}</div>;
```